src/cfg.rs

```diff
@@ -61,7 +61,7 @@ impl Default for Config {
             statsd: StatsdConfig::default(),
             trie_branching_factor: 16,
             l1_rpc_url: "".to_string(),
-            fc_network: FarcasterNetwork::Testnet,
```

Comment:
Make sure to update the config for the live nodes before this is merged.

Reply:
Yep, this is done